scripts/cc-cli/src/interfaces/index.ts

```diff
@@ -0,0 +1,4 @@
+// Auto-generated via `yarn polkadot-types-from-defs`, do not edit
+/* eslint-disable */
+
+// export * from './types';
```

Comment:
How is that different than https://github.com/gluwa/creditcoin/tree/dev/creditcoin-js/src/interfaces ?

Looks the same to me.